src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -191,10 +198,6 @@ public static double log10sumLog10(final double[] log10p, final int start, final
         return maxValue + (sum != 1.0 ? Math.log10(sum) : 0.0);
     }
 
-    private static final double LOG1MEXP_THRESHOLD = Math.log(0.5);
```

Comment:
What were these constants used for? could they be used in programs that still haven't been ported and you are now removing them?